src/store/state.js

```diff
@@ -2,6 +2,8 @@ export default {
   user: null,
   info: null,
   categories: [],
+  category: {},
```

Comment:
initial state for object is `null` (or `undefined`). Empty object means you have non empty data.